packages/react/src/LayoutExamples/KitchenSink.tsx

```diff
@@ -23,6 +24,7 @@ type KitchenSinkProps = React.HTMLAttributes<HTMLDivElement>
 export function KitchenSink(props: KitchenSinkProps) {
   return (
     <>
+      <DevGrid />
```

Comment:
I think you should add this as a global decorator, so that it can be used in every story.